contractsSol5/KyberMatchingEngine.sol

```diff
@@ -437,7 +467,7 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
             rates[i] = tData.tokenToEth.rates[i];
             splitValuesBps[i] = tData.tokenToEth.splitValuesBps[i];
             isFeePaying[i] = tData.tokenToEth.isFeePaying[i];
-            ids[i] = convertAddressToReserveId(address(reserveAddresses[i]));
```

Comment:
I think main issue is that we want to get rid of this. But looking back at it now, it doesn't seem too bad since we the loop already exists anyway to populate the other params.

Reply:
my issue is the extra storage read not the loop
if we handle 8 reserves
its extra 8 * 800 gas
while we had this value to start with. so makes no sense reading it here